quickstarts/snyk/config.yml

```diff
@@ -0,0 +1,45 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: snyk
+
+# Displayed in the UI (required)
+title: Snyk
+
+# Long-form description of the quickstart (required)
+description: |
+  The Snyk quickstart allows you to send application security vulnerabilities of your services into New Relic.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Application security vulnerabilities from within New Relic with this integration with Snyk.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Harry Kimpel
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - Snyk
+  - Application Security
+  - AppSec
+  - DevSecOps
+  - Vulnerabilities
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+#alerts:
+#  - condition-a
+#  - condition-b
+dashboards:
+  - snyk
+# Documentation references
+documentation:
+  - name: Installation docs
+    url: https://www.kimpel.com/post/snyk-to-newrelic-using-prometheus-exporter/
+    description: Installation and configuration instructions for Snyk and the Snyk quickstart.
+
```

Comment:
We need to link this up to the install plan as well
```suggestion
install:
  - third-party-snyk
  
```

Reply:
Hi @aswanson-nr, looking at the other config.yml files I think the syntax should be like that:

```
installPlans:
  - third-party-snyk
```

Hope this is correct. BR, Harry

Reply:
Whoops! my bad, you are correct!